index.html

```diff
@@ -20,19 +20,137 @@
+          </div> 
+        </section>
+      </article>
+      <article id="card-work-4" class="mobile-card">
+        <img class="card-snapshot" src="Images_Background/Snapshoot Portfolio 4_Art Printing.svg" alt="Project 4 Snapshot">
+        <section class="project-description-block">
+          <h2 class="card-title">Tonic</h2>
+          <ul class="project-role-detail">
+            <li class="project-role-client ">CANOPY</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>Back End Dev</li>
+            <li><i class="fa-solid fa-circle fa-2xs project-role-counter"></i></li>
+            <li>2015</li>
+          </ul>
+          <p class="project-description-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="project-language-list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+          </ul>
+          <div class="button-container">
+            <button class="see-project-button">See project</button>
+          </div>          
+        </section>
+      </article>
+    </section>
+
+    <section id="about-myself">
+      <h1 class="introduction-name principal-margin">About Myself</h1>
+      <p class="introduction-paragraph principal-margin">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="lets-connect">LET'S CONNECT</h2>
+      <ul class="social-icons social-icons-margin-bottom-about-myself">
+        <li><a href="https://github.com/HFG43"><img src="Images_Background/GitHub.svg" alt="GitHub Icon"></a></li>
+        <li><a href="https://www.linkedin.com/in/hern%C3%A1n-g%C3%BCemes-a440591b/"><img src="Images_Background/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/Fingers.svg" alt="Fingers Icon"></a></li>
+        <li><a href="https://twitter.com/HFG__43"><img src="Images_Background/Twitter.svg" alt="Twitter Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/M.svg" alt="M Icon"></a></li>
+      </ul>
+      <div class="button-container principal-margin">
+        <button class="see-project-button">Get my resume</button>
+      </div>
+      <ul class="about-myself-skills principal-margin">
+        <li class="arrow-skills"><p>Languages</p><i class="fa fa-angle-up"></i></li>
+          <ul class="tech-languages">
+            <li><img src="Images_Background/JS-logo.svg" alt="JavaScript logo"><p class="language-name">JavaScript</p></li>
+            <li><img src="Images_Background/html-logo.svg" alt="HTML logo"><p class="language-name">HTML</p></li>
+            <li><img src="Images_Background/css-logo.svg" alt="CSS logo"><p class="language-name">CSS</p></li>
```

Comment:
[OPTIONAL] Please use background here in order to highlight these list items as in Figma. 
hint : background:#F7F7F9